test/built-ins/Promise/prototype/finally/species-symbol.js

```diff
@@ -1,22 +0,0 @@
-// Copyright (C) 2017 V8. All rights reserved.
-// This code is governed by the BSD license found in the LICENSE file.
```

Comment:
why is this deleted?

Reply:
This test seems like a false positive. It doesn't seem like the assertions are relying on the the Promise subclass at all, but just returning what finally returns.

I wonder if the goal was checking something like `.finally.call(MyPromise, () => {})` or the **resolved** value of the finally call.

For the current text, I don't see why `p instanceof MyPromise` would ever be true and that's why the info metatag is ever so important in tests like this.

Reply:
@ljharb This test is great, but it is a 1:1 duplicate of [`subclass-species-constructor-resolve-count.js`](https://github.com/tc39/test262/blob/master/test/built-ins/Promise/prototype/finally/subclass-species-constructor-resolve-count.js).

Reply:
I opened #2693 so we can move this one forward.